aml-development/frontend
# fix(listing mgmt): show correct count for empty data

AMLOS-788

**Description**
Log into application using Big Brother

Go to Listing Management
Click on the List icon.
Click on the Pending Org Approval or Draft or where the value is zero or no records.
Table is displayed with 1 to 0
Suggestions: It could be 0 to 0 or nothing. Or text with "No results" check with Dev Lead.

**How to Test**
Pull this branch
Follow the steps in the description
Verify that instead of `1 to 0`, it shows `No results found` when the table is empty
